CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.2.1] - 2019-04-02
```

Comment:
Wouldn't this be 4.2.0?

Reply:
I know we don't have an established practice yet, but I expected that we'd make code changes first under code review, then update versions/edit changelog/create a release after merging the changes.